src/api.rs

```diff
@@ -1408,13 +1413,18 @@ impl ApiRequest {
         method: &Method,
         url: &str,
         auth: Option<&Auth>,
+        pipeline_env: String,
     ) -> ApiResult<Self> {
         debug!("request {} {}", method, url);
+        debug!("pipeline: {}", pipeline_env);
 
         let mut headers = curl::easy::List::new();
         headers.append("Expect:").ok();
         headers
```

Comment:
```rs
        match pipeline_env {
            Some(env) => {
                debug!("pipeline: {}", env);
                headers
                    .append(&format!("User-Agent: sentry-cli/{} {}", VERSION, env))
                    .ok();
            }
            None => {
                headers
                    .append(&format!("User-Agent: sentry-cli/{}", VERSION))
                    .ok();
            }
        }
```

Reply:
Otherwise you'll get empty `pipeline: ` logs and unnecessary whitespace in the header

Reply:
GitHub doesnt support suggestions on empty opines, so had to paste it manually. Should replace lines 1423:1428